### DISCUSSION TITLE
The Deserted Shrimp

### DISCUSSION THREAD

**Discussion by PhilippePerret** [prawnpdf/prawn#1317] [General] [answered]
Hello,

I would like to know why the Prawn project was abandoned. Are all efforts now focused on the hexaPDF gem?

Thank you in advance for any response you can provide.

Enjoy!

**Comment by pointlessone** [comment, marked as answer]
Prawn is not abandoned. Let's call it "stable". You can find more insight into situation reading my past status updates somewhere here.

hexaPDF is a separate project. I personally have ho relation to it. However, hexaPDF's maintainer was a great help for Prawn before he started hexaPDF and even after remain one of the most active members of Prawn community.

**Reply by PhilippePerret** [reply]
Thank you, Alexander.
